profiles/realtime/tuned.conf

```diff
@@ -29,6 +29,10 @@ isolated_cores_online_expanded=${f:cpulist_online:${isolated_cores}}
 assert2=${f:assertion:isolated_cores contains online CPU(s):${isolated_cores_expanded}:${isolated_cores_online_expanded}}
 
 # Assembly managed_irq
+# Make sure isolate_managed_irq is defined before any of the variables that
+# use it (such as regex_search_ternary) are defined, so that child profiles can set
```

Comment:
`s/regex_search_ternary/managed_irq/`

`regex_search_ternary` is a function.

Reply:
Thanks, good catch.  It was late :)